src/models/tag.spec.ts

```diff
@@ -6,6 +6,8 @@ import { strings } from 'locutus/php';
 
 chai.use(deepEqualInAnyOrder);
 describe('tag model', () => {
+  // write test for getSuggestedTags checking isPremium conditional
```

Comment:
Is this a TODO for this PR?

Reply:
shoot yeah. sorry i let that slide too. Adding that test now.

Reply:
Where did this get added?  I'm not sure I see it.